src/docs/10_about/10_about-the-project.adoc

```diff
@@ -3,17 +3,19 @@
 
 include::../_feedback.adoc[]
 
-docToolchain is used by many people.
-Since the main focus of docToolchain is technical documentation, it is mainly used for internal projects not visible to the public.
+=== History
+docToolchain is an open source project dedicated to automating the creation of technical documentation.
 
-Up until version 2.0.0, docToolchain had no static site generator and thus was used behind the scenes.
-This has changed now and open source projects which use docToolchain start to be visible.
+Before the project started, the founders had never heard of the term 'docs as code'. All they knew was that they were sick of keeping their architecture diagrams up to date by copying them from a UML tool to a word processor. Being lazy developers, they thought "There must be a better way to do this!". So they automated the diagram export, ditched their word processors, and started using a markup renderer. This enabled them to reference the diagrams from within the text and update them before rendering the document.
```

Comment:
I'll have ot think a bit about this section. The first version, I think was too personal, this version now sounds "too big"? "the founders"... have to think about it.